### PR TITLE
Fm km 4878 tooltip mvp

### PR DESCRIPTION
**Pull request by markacianfrani**
## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

**Comment by markacianfrani** [comment]
If you add a tooltip to some plain text like: 
```
Hello my name is <rux-tooltip message="yo">whatup</rux-tooltip>
```
the content gets cursor pointer, which makes me think I can click it. It also prevents me from being able to select/copy the content. Im thinking we should remove this. 

But there's a larger UX concern here...in this example, there should probably be some visual affordance that **whatup** is a tooltip that you can hover over. Icon maybe? or is this example bad juju? Going to raise this up with design to make sure we capture that in our docs.

File: packages/web-components/package.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
         "@floating-ui/dom": "~1.0.1",
         "@stencil/core": "~2.18.0",
         "date-fns": "~2.21.1",
-        "date-fns-tz": "~1.3.3"
+        "date-fns-tz": "~1.3.3",
+        "docker": "~1.0.0"
```
Comment (markacianfrani) [inline packages/web-components/package.json:48]: not sure if this was intentional?

Comment (FMorrison87) [reply packages/web-components/package.json:48]: D'oh. Fixed.

File: packages/web-components/src/components/rux-slider/test/index.html [hunk 2]
```diff
@@ -104,7 +104,7 @@ <h2>Prop/Slot Testing</h2>
             <script>
                 const log = document.getElementById('log')
                 const ticks = document.getElementById('ticks')
-                ticks.axisLabels = ['0', '25', '50', '75', '100']
+                ticks.axisLabels = ['Low', 'Med', 'High', 'Oh Noes!']
```
Comment (markacianfrani) [inline packages/web-components/src/components/rux-slider/test/index.html:107]: whats going on here?

Comment (kiley-mitti) [reply packages/web-components/src/components/rux-slider/test/index.html:107]: @markacianfrani @FMorrison87 OH NO! I'm sorry. 🤣 Jonathan and I were having a side discussion about how ticks work and get labeled in slider and I wrote up something and shared screen but didn't think I committed it! How embarrassing. Let me fix.

**Pull request closed without merging**